tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -68,16 +68,16 @@ internal static partial class ConfigurationKeys
         public const string TraceEnabled = "DD_TRACE_ENABLED";
 
         /// <summary>
-        /// Configuration key for enabling Standalone ASM, thus disabling APM tracing and its billing.
+        /// Configuration key for enabling or disabling the Tracer's debug mode.
         /// Default is value is false (disabled).
         /// </summary>
-        public const string AppsecStandaloneEnabled = "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED";
```

Comment:
So with this change, you're no longer going to support `DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED` at all. I assume we're ok with making that breaking change, because it's experimental? You're not going to upset customers that are currently using it (if any - you can check to see if they are)?

Reply:
Good catch. It might be worth reading this one as a fall back if `DD_APM_TRACING_ENABLED` is not set.